src/zenml/cli/stack.py

```diff
@@ -1171,14 +1193,22 @@ def import_stack(
         data = read_yaml(filename)
         cli_utils.declare(f"Using '{filename}' to import '{stack_name}' stack.")
 
-    # assert zenml version is the same
+    # assert zenml version is the same if force is false
     if data["zenml_version"] != zenml.__version__:
-        cli_utils.error(
-            f"Cannot import stacks from other ZenML versions. "
-            f"The stack was created using ZenML version "
-            f"{data['zenml_version']}, you have version "
-            f"{zenml.__version__} installed."
-        )
+        if ignore_version_mismatch:
+            cli_utils.warning(
+                f"The stack that will be installed is using ZenML version "
+                f"{data['zenml_version']}. You have version "
+                f"{zenml.__version__} installed. Some components might not "
+                "work as expected."
+            )
+        else:
+            cli_utils.error(
+                f"Cannot import stacks from other ZenML versions. "
+                f"The stack was created using ZenML version "
+                f"{data['zenml_version']}, you have version "
+                f"{zenml.__version__} installed."
```

Comment:
```suggestion
                f"Cannot import stacks from other ZenML versions. "
                f"The stack was created using ZenML version "
                f"{data['zenml_version']}, you have version "
                f"{zenml.__version__} installed. You can "
                retry using the `--ignore-version-mismatch` 
                flag. However, be aware that this might
                fail or lead to other unexpected behavior."
```

Reply:
Great addition!